openedx/core/lib/request_utils.py

```diff
@@ -220,7 +218,7 @@ def _get_expected_error_settings_dict():
     if _EXPECTED_ERROR_SETTINGS_DICT is not None:
         return _EXPECTED_ERROR_SETTINGS_DICT
 
-    expected_errors = getattr(settings, 'EXPECTED_ERRORS', None)
+    expected_errors = getattr(settings, 'IGNORED_ERRORS', None)
```

Comment:
Another example of a variable that wasn't updated.
```suggestion
    ignored_errors = getattr(settings, 'IGNORED_ERRORS', None)
```

Reply:
@robrap sure work on it and also do some changes in function which we need to changes.

Reply:
These changes are under process it will completed in my commit